libbeat/filter/condition.go

```diff
@@ -11,247 +11,308 @@ import (
 	"github.com/elastic/beats/libbeat/logp"
 )
 
+type RangeValue struct {
+	gte *float64
+	gt  *float64
+	lte *float64
+	lt  *float64
+}
+
+type EqualsValue struct {
+	Int uint64
+	Str string
+}
+
 type Condition struct {
-	Equals   map[string]EqualsValue
-	Contains map[string]string
-	Regexp   map[string]*regexp.Regexp
-	Range    map[string]RangeValue
+	equals   map[string]EqualsValue
+	contains map[string]string
+	regexp   map[string]*regexp.Regexp
+	rangexp  map[string]RangeValue
```

Comment:
The reason why the struct `Condition` looks like this is that in the current version you don't need to add an extra "condition" tag. For example:
```
- drop_fields:
    equals:
       http.code: 200
    fields: ["http.request_headers", "http.response_headers"]
```
In the current implementation you cannot use the same key in the condition as it's a map. I am not sure if it's better to have a list instead of a map. Currently I am doing AND between them, not OR.